### PR TITLE
chore: added newline to end of index.html

### LINKED ISSUE #6: fix index.html format
index.html has no newline at EOF

### PR DESCRIPTION
**Pull request by oldskoolfan**
Resolves #6

### CODE REVIEW

**Review by DevVoxel — APPROVED** [review]
Awesome Change

File: index.html [hunk 1]
```diff
@@ -15,4 +15,4 @@ <h1>CSCI 2479 Spring 2022 &mdash; Bios</h1>
     </nav>
     <footer>Copyright &copy; 2022 all rights reserved</footer>
 </body>
-</html>
```
Comment (ryvrook) [inline index.html:18]: Remove html, good job.

File: index.html [hunk 2]
```diff
@@ -15,4 +15,4 @@ <h1>CSCI 2479 Spring 2022 &mdash; Bios</h1>
...
\ No newline at end of file
+</html>
```
Comment (ryvrook) [inline index.html:18]: Added html.

**Review by DevVoxel — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (oldskoolfan) [reply index.html:18]: thanks!

**Pull request merged**